ansible/roles/test/files/ptftests/arista.py

```diff
@@ -175,14 +175,12 @@ def run(self):
                 if cmd == 'cpu_down':
                     last_lacppdu_time_before_reboot = self.check_last_lacppdu_time()
                     self.lacp_pdu_time_on_down.append(last_lacppdu_time_before_reboot)
-                if cmd == 'cpu_up' or collect_lacppdu_time:
+                if (cmd == 'cpu_going_up' or cmd == 'cpu_up') and self.collect_lacppdu_time:
                     # control plane is back up, start polling for new lacp-pdu
                     last_lacppdu_time_after_reboot = self.check_last_lacppdu_time()
-                    if last_lacppdu_time_after_reboot != last_lacppdu_time_before_reboot:
+                    if int(last_lacppdu_time_after_reboot) != int(last_lacppdu_time_before_reboot):
```

Comment:
Should it be > instead of "!="?

Reply:
Sure, that would be a better check. Updated this in latest commit. Thanks.